alkira/resource_alkira_list_global_cidr.go

```diff
@@ -42,6 +42,12 @@ func resourceAlkiraListGlobalCidr() *schema.Resource {
 				Required: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"tags": {
+				Description: "A list of service types associated with the GlobalCidrList.",
```

Comment:
Just say `A list of associated service types` instead of repeating the list?

Reply:
Thanks